packages/teams-js/src/internal/interfaces.ts

```diff
@@ -50,3 +52,19 @@ export interface DOMMessageEvent {
   func: string;
   args?: any[];
 }
+
+/**
+ * @internal
+ * Limited to Microsoft-internal use
+ *
+ * UUID object
+ */
+export class UUID {
```

Comment:
This started out as an interface which is why it is in the `interfaces.ts` file, but now that it is a class with functionality it would probably make sense to give it its own file and test file